packages/grafana-data/src/valueFormats/arithmeticFormatters.ts

```diff
@@ -2,21 +2,21 @@ import { DecimalCount } from '../types/displayValue';
 
 import { toFixed, FormattedValue } from './valueFormats';
 
-export function toPercent(size: number, decimals: DecimalCount): FormattedValue {
+export function toPercent(size: number | null, decimals: DecimalCount): FormattedValue {
   if (size === null) {
     return { text: '' };
   }
   return { text: toFixed(size, decimals), suffix: '%' };
 }
 
-export function toPercentUnit(size: number, decimals: DecimalCount): FormattedValue {
+export function toPercentUnit(size: number | null, decimals: DecimalCount): FormattedValue {
```

Comment:
maybe silly.. do you want to pull these changes into its own PR?  IIUC, this is just improving the types and tests -- and is not actually part of the StatsPanel changes

Reply:
I was thinking the same thing, actually. Should have just done that. On it.

Reply:
removed.